master/internal/db/postgres_experiments.go

```diff
@@ -355,10 +355,10 @@ SELECT t.id FROM (
   SELECT t.id,
     %s((v.metrics->'validation_metrics'->>$1)::float8) as best_metric
   FROM trials t
-  JOIN validations v ON t.id = v.trial_id
+  LEFT JOIN validations v ON t.id = v.trial_id
   WHERE t.experiment_id=$2
   GROUP BY t.id
-  ORDER BY best_metric %s
+  ORDER BY best_metric %s NULLS LAST
```

Comment:
probably not worth doing training length as a fallback as a part of this PR, but does a secondary sort make sense here? 

maybe `end_time DESC NULLS FIRST` to see the active ones?